lib/handlers/put.js

```diff
@@ -6,12 +6,40 @@ const getContentType = require('../utils').getContentType
 const HTTPError = require('../http-error')
 const { stringToStream } = require('../utils')
 
+// TODO: ask alain a better way to get the suffix variables here
+const RESERVED_SUFFIXES = ['.acl', '.meta']
+
+/**
+ * This function is used to make sure a resource or container which contains
+ * reserved suffixes for auxiliary documents cannot be created.
```

Comment:
```suggestion
 * This function is used to prevent creation of a resource or container
 * which contains auxiliary documents named with reserved suffixes.
```